File: backend/src/main/java/top/vuhe/mapper/UserMapper.java
package top.vuhe.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import org.apache.ibatis.annotations.Mapper;
import org.springframework.stereotype.Repository;
import top.vuhe.entity.auth.User;

/**
 * @author zhuhe
 */
@Mapper
@Repository
public interface UserMapper extends BaseMapper<User> {
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/mapper/UserMapper.kt
package top.vuhe.sw.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import org.apache.ibatis.annotations.Mapper
import org.springframework.stereotype.Repository
import top.vuhe.sw.entity.auth.User

@Mapper
@Repository
interface UserMapper: BaseMapper<User>